src/options/imports/index_ops.js

```diff
@@ -0,0 +1,56 @@
+import index from 'src/search/search-index'
+
+const db = index.db._db
```

Comment:
The stuff in `src/search/search-index` actually all lives in the background script. Importing it here will duplicate that code into the options app script as well. Not your fault; it's not entirely clear. I will have to set up a simple two-method interface from the bg script for this code to call.